common/export-common-cartridge-util/pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<modelVersion>4.0.0</modelVersion>
+<parent>
+	<groupId>org.sakaiproject</groupId>
+	<artifactId>master</artifactId>
```

Comment:
It's common practice in maven to extend the closest POM so that local section like dependency management can be used closest to where it makes sense.